code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/github/SetProjectMembersActionHandler.java

```diff
@@ -0,0 +1,50 @@
+package io.cattle.platform.iaas.api.auth.github;
+
+import io.cattle.platform.api.action.ActionHandler;
+import io.cattle.platform.api.auth.ExternalId;
+import io.cattle.platform.api.auth.Policy;
+import io.cattle.platform.core.constants.ProjectConstants;
+import io.cattle.platform.core.model.Account;
+import io.cattle.platform.iaas.api.auth.dao.AuthDao;
+import io.github.ibuildthecloud.gdapi.context.ApiContext;
+import io.github.ibuildthecloud.gdapi.exception.ClientVisibleException;
+import io.github.ibuildthecloud.gdapi.request.ApiRequest;
+import io.github.ibuildthecloud.gdapi.util.ResponseCodes;
+
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+import javax.inject.Inject;
+
+public class SetProjectMembersActionHandler implements ActionHandler{
+
+    @Inject
+    AuthDao authDao;
+
+    @Inject
+    ProjectMemberResourceManager projectMemberResourceManager;
+
+    @Override
+    public Object perform(String name, Object obj, ApiRequest request) {
+        Policy policy = (Policy) ApiContext.getContext().getPolicy();
+        Account account = (Account) obj;
+        account = authDao.getAccountById(account.getId());
+        if (account == null) {
+            throw new ClientVisibleException(ResponseCodes.NOT_FOUND);
+        }
+        if (!authDao.isProjectOwner(account.getId(), policy.getAccountId(),
+                policy.isOption(Policy.AUTHORIZED_FOR_ALL_ACCOUNTS), policy.getExternalIds())){
+            throw new ClientVisibleException(ResponseCodes.FORBIDDEN, "Forbidden", "You must be a project owner to change members", null);
+        }
+        LinkedHashMap<String, Object> reqObj = (LinkedHashMap<String, Object>) request.getRequestObject();
```

Comment:
We could but the type that getRequestObject() returns is Object so... Not sure what to do in this instance.